backend/.env.example

```
@@ -0,0 +1,2 @@
+PORT=3000
+CORS_ORIGIN=http://localhost:5173
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)